src/components/common/ColonyHome/hooks/useDashboardHeader.ts

```diff
@@ -44,17 +45,29 @@ export const useDashboardHeader = (): ColonyDashboardHeaderProps => {
   const { colony, colonySubscription } = useColonyContext();
   const { pathname } = useLocation();
   const colonyUrl = `${window.location.host}${pathname}`;
-  const { handleClipboardCopy, isCopied } = useCopyToClipboard(5000);
   const {
-    handleClipboardCopy: itemHandleClipboardCopy,
-    isCopied: itemIsCopied,
+    handleClipboardCopy: shareUrlhandleClipboardCopy,
```

Comment:
could we rename all of these to something like `handleShareUrlCopy`

Reply:
Made this change! :)